packages/moertel/src/mortar/mrtr_manager.H

```diff
@@ -610,6 +610,31 @@ public:
   */
   const Epetra_CrsMatrix* M() const { return M_.get();}
 
+  std::map<int,int> lm_to_dof()
+  {
+      std::map<int,Teuchos::RCP<MOERTEL::Interface> >::iterator curr;
+      // Create a map from Lagrange multiplier dofs to the primal dofs on the same node
+      std::vector<MOERTEL::Node*> nodes;
+      std::map<int,int> lm_to_dof;
+      for (curr=interface_.begin(); curr!=interface_.end(); ++curr)
```

Comment:
We have C++11; use a range-based for loop:
```cpp
for (auto&& keyval : interface) {
  RCP<MOERTEL::Interface> inter = keyval.second;
  // ...
}
```

Reply:
Thanks for highlighting this improvement! 
I will do so.